### ISSUE TITLE
Cannot deploy uploaded form with label translation columns

### ISSUE THREAD

**Issue by jnm** [kobotoolbox/kpi#852]
See [singlesheettrans.xlsx](https://github.com/kobotoolbox/kpi/files/400440/singlesheettrans.xlsx), which imports directly into KC without issue but causes the following on KPI:

```
Traceback:
[snip]
File "/home/ubuntu/k/kpi/kpi/deployment_backends/mixin.py" in connect_deployment
  12.                 DEPLOYMENT_BACKENDS[backend](self).connect(**kwargs)
File "/home/ubuntu/k/kpi/kpi/deployment_backends/kobocat_backend.py" in connect
  249.         csv_io = self.to_csv_io(self.asset.to_versioned_xls_io(), id_string)
File "/home/ubuntu/k/kpi/kpi/models/asset.py" in to_versioned_xls_io
  172.             overwrite_settings=True
File "/home/ubuntu/k/kpi/kpi/models/asset.py" in to_xls_io
  147.                     _add_contents_to_sheet(cur_sheet, ss_dict[sheet_name])
File "/home/ubuntu/k/kpi/kpi/models/asset.py" in _add_contents_to_sheet
  110.                     for key in row.keys():

Exception Type: AttributeError at /assets/amVTZQRrk4kwYaodiptUio/deployment/
Exception Value: 'unicode' object has no attribute 'keys'
```

Problem affects 263387f (current `develop`) but not 64be693 (current PROD)

**Comment by dorey** [comment]
This is because `formpack.utils.flatten_content` does not remove the `translations: [...]` key from the content. A fix for this could be to put a `_flattened_content.pop('translations', None)` in `XlsExportable.valid_xlsform_content()`

**Comment by jnm** [comment]
@dorey, that does make the form deployable, but the resulting form on KC has a single `label` column with the translations concatenated:

| $kuid     | required | type    | name    | label                |
|-----------|----------|---------|---------|----------------------|
| 7MuwxxsxD | true     | integer | tacoint | ¿Cuál es tu taco?Hmm |

Whereas the [original](https://github.com/kobotoolbox/kpi/files/400440/singlesheettrans.xlsx) looked like:

| label::Español | label | type | name |
|-------------------|-------|---------|---------|
| ¿Cuál es tu taco? | Hmm | integer | tacoint |

**Comment by dorey** [comment]
I will see if I can fix this by using formpack#100 to avoid needing to call `"valid_xlsform_content()` in as many places.

**Comment by jnm** [comment]
Looks like @dorey fixed this. Cannot reproduce any longer.
